src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/MicrosoftIdentityPlatform/MicrosoftIdentityPlatformApplicationManager.cs

```diff
@@ -386,56 +444,55 @@ internal static bool UpdateImplicitGrantSettings(Application app, ProvisioningTo
             return needsUpdate;
         }
 
-        private async Task AddApiPermissionFromBlazorwasmHostedSpaToServerApi(
-            GraphServiceClient graphServiceClient,
-            Application createdApplication,
-            ServicePrincipal? createdServicePrincipal,
-            bool isB2C)
-        {
-            if (createdServicePrincipal is null)
-            {
-                throw new ArgumentNullException(nameof(createdServicePrincipal));
-            }
-
-            var requiredResourceAccess = new List<RequiredResourceAccess>();
-            var resourcesAccessAndScopes = new List<ResourceAndScope>
-                {
-                    new ResourceAndScope($"api://{createdApplication.AppId}", DefaultProperties.ApiScopes)
-                    {
-                         ResourceServicePrincipalId = createdServicePrincipal.Id
-                    }
-                };
-
-            await AddPermission(
-                graphServiceClient,
-                requiredResourceAccess,
-                resourcesAccessAndScopes.GroupBy(r => r.Resource).First()).ConfigureAwait(false);
-
-            Application applicationToUpdate = new Application
-            {
-                RequiredResourceAccess = requiredResourceAccess
-            };
-
-            await graphServiceClient.Applications[createdApplication.Id]
-                .Request()
-                .UpdateAsync(applicationToUpdate).ConfigureAwait(false);
-
-            if (isB2C)
-            {
-                var oAuth2PermissionGrant = new OAuth2PermissionGrant
-                {
-                    ClientId = createdServicePrincipal.Id,
-                    ConsentType = "AllPrincipals",
-                    PrincipalId = null,
-                    ResourceId = createdServicePrincipal.Id,
-                    Scope = DefaultProperties.ApiScopes
-                };
-
-                await graphServiceClient.Oauth2PermissionGrants
-                    .Request()
-                    .AddAsync(oAuth2PermissionGrant).ConfigureAwait(false);
-            }
-        }
+        //private async Task AddApiPermissionFromBlazorwasmHostedSpaToServerApi(
```

Comment:
Let's remove the commented out method. If we need part of this later, we can come back to this PR.